index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" style="width: 20em;" alt="">
+      <nav id="nav-bar" >
+          <a class="nav-link" href="#header">Features</a>
+          <a class="nav-link" href="#video">How It Works</a>
+          <a class="nav-link" href="#header">Pricing</a>
+      </nav>
+    </header>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input id="email" type="email" placeholder="Enter Email" name="email">
+        <input id="submit"  type="submit" name="submit">
+    </form>
+
+   <div class="video">
+    <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" 
+    title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </div>
+   
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
```

Comment:
Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.